### PR TITLE
Initial inclusion of IMPACT model

### PR DESCRIPTION
**Pull request by bgsandan**
Tables designed to reflect output csv files from IMPACT model to aid in updating.  Data import sql for test data repo auto-generated from csv

Adds table for storing main projection data and dictionaries for scenarios and commodity names

### CODE REVIEW

**Comment by rbroth** [comment]
The filenames for table creation have an extra period that's causing flyway to not see them

**Comment by rbroth** [comment]
Do you want me to walk you through installing flyway on your local machine so you can test changes without having to commit them? I've got it setup that way and it's very convenient

File: db/dictionary-data/V0.2.0.600__impact-data.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
+INSERT INTO
+    impact_scenario (name, description )
```
Comment (rbroth) [inline db/dictionary-data/V0.2.0.600__impact-data.sql:2]: be explicit regarding what columns get stuff inserted

```suggestion
    impact_scenario (id, name, description )
```

Comment (bgsandan) [reply db/dictionary-data/V0.2.0.600__impact-data.sql:2]: :tired_face: Doh!  Good spot.  Updated now

Comment (bgsandan) [reply db/dictionary-data/V0.2.0.600__impact-data.sql]: :tired_face: missed that one.  Silly copy&paste error.  Now fixed

File: db/objects/impact/V.0.1.0.70__impact-food-availability.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
+CREATE TABLE IMPACT_FOOD_AVAILABILITY (
+	ssp                 		text REFERENCES IMPACT_SCENARIO (ID)
+	, comdty            		text REFERENCES IMPACT_COMMODITY (ID)
+	, cty       				text REFERENCES COUNTRY (ID)
+	, yr          				smallint
```
Comment (rbroth) [inline db/objects/impact/V.0.1.0.70__impact-food-availability.sql]: We can use full words, the letter shortage has ended :P 

Also, I've noticed that the code is mixing tabs and spaces, you may want to check your editor settings

```suggestion
	, commodity                 text REFERENCES IMPACT_COMMODITY (ID)
	, country                   text REFERENCES COUNTRY (ID)
	, year                      smallint
```

**Pull request merged**